README.md

```diff
@@ -1,2 +1,89 @@
-# recombee
-Recombee API client.
+# Recombee
+
+A [Recombee API](https://docs.recombee.com/api.html) client written in Go. 
+
+[![GoDoc][GoDoc-Image]][GoDoc-Url]
+
+[GoDoc-Url]: https://pkg.go.dev/github.com/nats-io/nats.go
+[GoDoc-Image]: https://img.shields.io/badge/GoDoc-reference-007d9c
+
+## Features
+
+Is capable of managing:
```

Comment:
Should be capable of everything in my opinion. Making new PR makes the documentation either old or have to be updated with it. However we dont support sometimes all of the calls on each entity. 
Please refer to code and docs that we cover it fully. If not this section is useless.

Reply:
This is the state we are at, it's good practice to mention that not all API endpoints are covered. Furthemore recombee may decide to add new endpoints and before we notice it, the client will be out of sync. It's a common practice to pinpoint exactly what the features are.

Reply:
I am able to approve it only if you check that we cover all of the mentioned sections fully, check it one more time.
1. Items, we do not implement `Delete More Items`
2. Item properies, we do not implement `List Item Properties`
3. Item properties, we do not implement the `Update More Items` call etc.

Please be so kind, go through official docs and do not say that it is feature that we implemented as it is not fully covered. 

I rather do not include it or include general case, that we do not fully cover the features. 

I would focus on implementing the calls in next PR and update docs of features once it is fully covered.

Reply:
Nicely done